docs/en/jpql-with-kotlin-jdsl/paths.md

```diff
@@ -13,6 +13,75 @@ entity(Book::class, "b").path(Book::isbn).path(Isbn::value)
 entity(Book::class, "b")(Book::isbn)(Isbn::value)
 ```
 
+## Java entity
+
+`path()` and `invoke()` take `KProperty1` or `KFuction1` as an argument.
+`KFunction1` is useful when you use Java entity with private property and public getter.
+
+```java
+@Entity
+public class Book {
+    @Id
+    private Long id;
+
+    private String title;
+
+    public String getTitle() {
+        return title;
+    }
+}
+```
+
+```kotlin
+// compile error
+path(Book::title)
+
+// Book.title
+path(Book::getTitle)
+```
+
+Kotlin JDSL follows the following rules to infer property name from getter name.
```

Comment:
It would be nice to change the sentence to something like this: `Kotlin JDSL infers the property name from the getter with the following rules:`